site/docs/api/serialization.md

```diff
@@ -292,5 +314,8 @@ hazelcast:
 ```
 
 All the classes - data types, serializers & hooks - should be present
-on the server classpath, ideally in server's `lib` directory packaged as
-a jar file.
+on the cluster classpath, ideally in server's `lib` directory packaged as
+a jar file. Moreover, used type ids have to be unique across all
+serializers.
+Despite those limitations cluster level serializers offer
+full support for IMDG [sources and sinks](sources-sinks.md).
```

Comment:
we should mention this is required for supporting querying, projections and so on.